Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-01-23
Initial Package Version: XFree 4.2.1
Upstream Status: Unknown
Origin: Gentoo
Description: If building servers and SharedLibXau is defined, 
	always build libXau by adding it to EXTRASYSLIBS

--- xc/programs/Xserver/Imakefile.orig	2002-12-01 19:07:44.000000000 +0200
+++ xc/programs/Xserver/Imakefile	2002-12-01 19:10:26.000000000 +0200
@@ -276,8 +276,16 @@
             USB = UsbHidLib
 #endif
 #endif
 #ifdef ServerExtraSysLibs
+# if SharedLibXau
+   EXTRASYSLIBS = ServerExtraSysLibs $(XAUTHLIB)
+# else
    EXTRASYSLIBS = ServerExtraSysLibs
+# endif
+#else
+# if SharedLibXau
+   EXTRASYSLIBS = $(XAUTHLIB)
+# endif
 #endif
 #if HasPam && HasPamMisc
         PAMLIBS = PamLibraries PamMiscLibraries
